configure.ac

```diff
@@ -1231,18 +1231,18 @@ else
 
   if test -n "$SCILAB"; then
     # Check for Scilab version (needs api_scilab so needs version 5.3.3 or higher)
-    SCILAB_FULL_VERSION=`$SCILAB -version | head -1 | sed -e 's|Scilab version \"\(.*\)\"|\1|g'`
+    SCILAB_FULL_VERSION=`$SCILAB -nwni -version | head -1 | sed -e 's|Scilab version \"\(.*\)\"|\1|g'`
```

Comment:
It seems unnecessarily confusing that `SCILAB_FULL_VERSION` here becomes `SCILAB_VERSION` in the makefiles under `Examples`, but there's also a `SCILAB_VERSION` here which is different (if I follow, the former is e.g. `6.0.1` while the latter is `601`).

I'd suggest renaming the `SCILAB_VERSION` here to something like `SCILAB_VERSION_NO_DOTS` and then `SCILAB_FULL_VERSION` here can just be `SCILAB_VERSION`.

Reply:
ok I will rename